src/laser_core/migration.py

```diff
@@ -33,43 +33,87 @@
 
 def gravity(pops: np.ndarray, distances: np.ndarray, k: float, a: float, b: float, c: float, **kwargs):
     r"""
-    Calculate a gravity model network with an optional maximum export fraction constraint.
-    This function computes a gravity model network based on the provided populations
-    and distances, and, if specified, then normalizes the rows of the resulting network matrix
-    such that no row exceeds the specified maximum export fraction.
-
-    Mathematical formula:
-
-        element-by-element: :math:`network_{i,j} = k \times p_i^a \times p_j^b / distance_{i,j}^c`
-
-        as implemented in NumPy math:
-
-            ``network = k * (pops[:, np.newaxis] ** a) * (pops ** b) * (distances ** (-1 * c))``
-
-    Parameters:
-
-        pops (numpy.ndarray): 1D array of populations.
-        distances (numpy.ndarray): 2D array of distances between the populations.
-        k (float): Scaling constant.
-        a (float): Exponent for the population of the origin.
-        b (float): Exponent for the population of the destination.
-        c (float): Exponent for the distance.
-        \*\*kwargs: Additional keyword arguments (not used in the current implementation).
-
-    Returns:
-
-        numpy.ndarray: A matrix representing the gravity model network with rows optionally normalized to respect the maximum export fraction.
+    Calculate a gravity model network.
+
+    This function computes a gravity model network based on the provided populations and distances.
+    The gravity model estimates migration or interaction flows between populations using a mathematical formula
+    that incorporates scaling, population sizes, and distances.
+
+    **Mathematical Formula**:
+
+    .. math::
+        network_{i,j} = k \cdot \frac{p_i^a \cdot p_j^b}{distance_{i,j}^c}
+
+    As implemented in NumPy:
+
+    .. code-block:: python
+
+        network = k * (pops[:, np.newaxis] ** a) * (pops ** b) * (distances ** (-1 * c))
+
+    **Parameters**:
+        pops (numpy.ndarray):
+            1D array of population sizes for each node.
+        distances (numpy.ndarray):
+            2D array of distances between nodes. Must be symmetric, with self-distances (diagonal) handled.
+        k (float):
+            Scaling constant to adjust the overall magnitude of interaction flows.
+        a (float):
+            Exponent for the population size of the origin node.
+        b (float):
+            Exponent for the population size of the destination node.
+        c (float):
+            Exponent for the distance between nodes, controlling how distance impacts flows.
+        **kwargs:
```

Comment:
Asterisks here need to be escaped to keep from getting a doc build warning (warnings are errors when RtD builds the official docs).

`\*\*kwargs`